allure-generator/webpack.config.js

```diff
@@ -35,17 +33,26 @@ module.exports = (env) => {
                 test: /\.(png|svg|woff2?|ttf|eot)(\?.*)?$/,
                 use: 'url-loader'
             }, {
-                test: /\.css$/,
+                test: /\.scss$/,
                 use: development
-                    ? ['style-loader', 'css-loader', postcssLoader]
+                    ? ['style-loader', 'css-loader', postcssLoader, 'sass-loader']
                     : ExtractTextPlugin.extract({
                         fallback: 'style-loader',
                         use: [
                             'css-loader',
-                            postcssLoader
+                            postcssLoader,
+                            'sass-loader'
                         ]
                     })
             }, {
+                test: /\.css$/,
```

Comment:
we still support plain css files, that are imported from libraries (e.g. font-awesome)